src/public/index.html

```diff
@@ -60,6 +60,18 @@ <h1 id="rpsStat">-</h1>
       <canvas id="rpsChart" width="200" height="100"></canvas>
     </div>
   </div>
+  <div class="container">
+    <div class="stats-column">
+      <h5>Status Codes</h5>
+      <h6 style="color: #75D701">2xx</h6>
+      <h6 style="color: #47b8e0">3xx</h6>
+      <h6 style="color: #ffc952">4xx</h6>
+      <h6 style="color: #E53A40">5xx</h6>
```

Comment:
Inline styles are bad, I know, I'm lazy. Also, i think there's better solution to unify them with these in JS file